src/cloud/components/atoms/MarkdownView/index.tsx

```diff
@@ -156,6 +157,9 @@ const MarkdownView = ({
       createElement: React.createElement,
       Fragment: React.Fragment,
       components: {
+        img: ({ src }: any) => {
```

Comment:
@Rokt33r this creates div and other elements iniside parent <p> element and react complains about it, is there a way to have it like this but to avoid the p -> div nesting? (i.e. can we remove parent <p> element?)

If we don't want this to complain and there is no way to do it like this, then I suppose the element should be only img and it should fire some event to enlarge particular img (src URL), and then we could create other elements outside?

Reply:
We can use span instead of div

Reply:
I think we can, but there are multiple divs defined inside, and making them span elements behave differently.

Also, the difference:
https://stackoverflow.com/questions/3890172/span-with-display-block

Reply:
Add second commit with spans instead od divs, have a look, let me know if there are any problems.

Tested all functionality with this changes

Reply:
Why don't we just have a single modal api to display a single image in a modal view rather than implementing enlarge view for every image?

Reply:
Not sure what you mean by this, It is extracted out inside a component which shows the image only. When user clicks on enlarge it shows a 'modal' which shows the image. 

Yes, on the outset, we could just replace this with some modal dialog which would receive necessary data and show the image, I would still need to create separate component for each image, its actions and bind the action of enlarge to send the data to the modal.